1-mandatory/1-writers.js

```diff
@@ -62,7 +67,14 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let deadWriter = writers.filter(
+    (writer) => !writer.alive && writer.age >= 40 && writer.age <= 49
+  );
+  deadWriter.forEach((writer) =>
```

Comment:
you don't need to do `let deadWriter` you can chain together array methods like `writers.filter(...).forEach(...)`